modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -261,6 +256,21 @@ export default class Tile3DLayer extends CompositeLayer {
     );
   }
 
+  _takeNecessaryAttributes(contentAttributes) {
```

Comment:
- Make a pure function instead of class method.
- The name should reflect that it's for mesh generation.

Reply:
Done